src/index.js

```diff
@@ -2,14 +2,21 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
+import { store } from "./store";
 import { makeServer } from "./server";
-
+import { BrowserRouter } from "react-router-dom";
+import { Provider } from "react-redux";
 // Call make Server
 makeServer();
+console.log(store.getState())
```

Comment:
remove console logs. You can try the redux devtools extension for keeping a track of the states.